main.cpp

```diff
@@ -1,15 +1,161 @@
 #include <getopt.h>
 #include <iostream>
+#include <string>
 #include <cstdlib>
+// #include <cstdio>
+// #include <stdexcept>
 
 #include "helpers.h"
 
-using namespace std;
+// using namespace std;
```

Comment:
This too!

Reply:
@Ank1taS Please resolve this also I don't know about the header comments you have added, we can discuss later about that.